feat(prisma-client): new nextjs-starter-webpack example (#8175)
prisma/prisma-examples: prisma-postgres/nextjs-starter-webpack/README.md

## Tech Stack

- Next.js 15
  - Runtime: Node.js 20.19.0
  - Bundler: Webpack 5
- ESM
  - `package.json` contains `{ "type": "module" }`
  - `next.config.js` -> `next.config.mjs`
  - `postcss.config.js` -> `postcss.config.mjs`
- Prisma Client with the `prisma-client` generator
  See the [Prisma schema file](./prisma/schema.prisma) for details.
  
  ```prisma
  generator client {
    provider = "prisma-client"
    output = "../lib/.generated/prisma"
    previewFeatures = ["driverAdapters", "queryCompiler"]
    runtime = "nodejs"
  }
  ```

## Getting started

### 1. Clone the repository

Clone the repository, navigate into it and install dependencies:

```
git clone git@github.com:prisma/prisma-examples.git --depth=1
cd prisma-examples/prisma-postgres/nextjs-starter-webpack
npm install
```

### 2. Configure environment variables

Create a `.env` in the root of the project directory:

```bash
touch .env
```

Now, open the `.env` file and set the `DATABASE_URL` environment variables with the values of your connection string and your Prisma Postgres connection string:

```bash
# .env

# Prisma Postgres connection string (used for migrations)

> MIGRATE_DATABASE_URL="__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__"

Author: ```suggestion
DATABASE_URL="__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__"
```

# Postgres connection string (used for queries by Prisma Client)

> DIRECT_DATABASE_URL="__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__"

Author: ```suggestion
DIRECT_URL="__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__"
```

NEXT_PUBLIC_URL="http://localhost:3000"
```

> Note that `__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres + Accelerate connection string. Notice that the Accelerate connection string has the following structure: `postgres://<username>:<password>@<host>:<port>/<database>?sslmode=require`.

Reviewer: ```suggestion
Note that `__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres + Accelerate connection string. Notice that the Accelerate connection string has the following structure: `prisma+postgres://accelerate.prisma-data.net/?api_key=<api_key_value>`.
```
